### PR TITLE
Configurable port number for services in demo

### PR DESCRIPTION
**Pull request by allenlsy**
**Description**:

Resolves issue https://github.com/openservicemesh/osm/issues/2755

Support configuring the port number for bookstore and bookwarehouse services in the demo. They can be configured using the `BOOKSTORE_PORT` and `BOOKWAREHOUSE_PORT` in `.env`.

Manual tested by customizing port number to `12001` in `.env` . The demo works as expected.

Screen Shot 2021-06-17 at 2 09 15 PM

Screen Shot 2021-06-17 at 2 08 33 PM

Note: The ports for bookstore-v1 and bookstore-v2 services are not distinguished. They both use BOOKSTORE_PORT.

**Affected area**:
| Functional Area            |     |
| -------------------------- | --- |
| New Functionality          | [ ] |
| Documentation              | [ ] |
| Install                    | [ ] |
| CLI Tool                   | [ ] |
| Certificate Management     | [ ] |
| Control Plane              | [ ] |
| Ingress                    | [ ] |
| Egress                     | [ ] |
| Networking                 | [ ] |
| Observability              | [ ] |
| SMI Policy                 | [ ] |
| Sidecar Injection          | [ ] |
| Security                   | [ ] |
| Upgrade                    | [ ] |
| Tests                      | [ ] |
| CI System                  | [ ] |
| Demo                       | [X] |
| Performance                | [ ] |
| Other                      | [ ] |

Please answer the following questions with yes/no.

1. Does this change contain code from or inspired by another project?
    -   Did you notify the maintainers and provide attribution?

No

1. Is this a breaking change?

No

### CODE REVIEW

**Review by shashankram — CHANGES_REQUESTED** [review]
I saw the issue https://github.com/openservicemesh/osm/issues/2755 and we settled on port 14001 (a pretty unique port that's not going to be commonly used elsewhere). I don't see why we are trying to introduce more complexity in the automated demo with trying to make the port configurable.

I am inclined to close https://github.com/openservicemesh/osm/issues/2755 unless there is a reason it is absolutely needed.

**Comment by allenlsy** [comment]
@ksubrmnn is this the reason we want make it configurable?

https://github.com/openservicemesh/osm/pull/2583#issuecomment-791749920

**Comment by allenlsy** [comment]
Closing the PR, since we haven't mutually agreed working on it.

File: .env.example [hunk 1]
```diff
@@ -43,6 +43,14 @@ export BOOKSTORE_NAMESPACE=bookstore
 # This cannot be the default namespace because it has to be a namespace that can be deleted.
 # Default: bookwarehouse
 export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
+
```
Comment (snehachhabria) [inline .env.example:46]: should the bookthief port all be configured ?

**Review by snehachhabria — APPROVED** [review]

**Comment by allenlsy** [comment]
Closing the PR as the feature is not required, and we want to keep our demo app simple.

**Pull request closed without merging**